boards/hifive1/board.c

```diff
@@ -96,7 +96,7 @@ void board_init_clock(void)
      /* Note: The range is limited to ~100MHz and depends on PLL settings */
     PRCI_set_hfrosctrim_for_f_cpu(CPU_DESIRED_FREQ, PRCI_FREQ_UNDERSHOOT);
 
-     /* disable uncrustify *INDENT-OFF* */
+     /* begin{code-style-ignore} */
```

Comment:
Uncrustify moves this comment one space to the left. Why not fix it ;-)? (the code below probably needs to be moved as well though :-/)

Reply:
> Why not fix it ;-)?

unrelated! ;)